src/Sentry/Internal/PlatformEventProcessor.uwp.cs

```diff
@@ -0,0 +1,82 @@
+#if WINDOWS_UWP
+using System;
+using Sentry.Extensibility;
+using Windows.ApplicationModel;
+using Windows.Security.ExchangeActiveSyncProvisioning;
+using Windows.System.Profile;
+
+namespace Sentry.Internal
+{
+
+    internal class PlatformEventProcessor : ISentryEventProcessor
+    {
+        private Lazy<UwpContext> _uwpContext = new Lazy<UwpContext>(() => new UwpContext());
+        private volatile bool _uwpContextLoaded = true;
+        private class UwpContext
+        {
+            public string DeviceFamily { get; set; }
+            public string DeviceManufacturer { get; set; }
+            public string DeviceModel { get; set; }
+            public string DeviceFriendlyName { get; set; }
+            public string OsName { get; set; }
+            public string OsVersion { get; set; }
+            public string OsArchitecture { get; set; }
+
+
+            public UwpContext()
+            {
+
+                DeviceFamily = AnalyticsInfo.VersionInfo.DeviceFamily;
+
+                ulong version = ulong.Parse(AnalyticsInfo.VersionInfo.DeviceFamilyVersion);
+                ulong major = (version & 0xFFFF000000000000L) >> 48;
```

Comment:
protocol's `debug_meta` actually uses the individual parts: https://develop.sentry.dev/sdk/event-payloads/debugmeta/#attributes

We'll need that if this needs `pdb` server side symbolication which I don't think we support just yet (for .NET Native)